src/main/kotlin/com/ampnet/reportserviceeth/service/data/CurrencyFormatting.kt

```diff
@@ -3,9 +3,7 @@ package com.ampnet.reportserviceeth.service.data
 import java.math.BigInteger
 import java.text.DecimalFormat
 
-const val FROM_CENTS_TO_EUROS = 100.0
+const val DECIMALS_PRECISION = 1_000_000_000_000_000_000
```

Comment:
Is this value present in Web3j library?

Reply:
There is org.web3j.utils.Convert class with units but it uses BigDecimal which only unnecessarily complicates the calculation. I would leave it as is.